README.md

```diff
@@ -116,31 +116,17 @@ Migrations are automatically applied in the pytest runs, so you needn't run them
 ### Using the benchmark fixtures
 
 We have a number of pytest fixtures defined which can be used to automatically track certain metrics in the benchmark database.
-They are summarized here:
+The most relevant ones are summarized here:
 
-**`benchmark_db_engine`**: The SQLAlchemy engine for the benchmark sqlite database. You can control the database name with the environment variable `DB_NAME`, which defaults to `benchmark.db`. Most tests shouldn't need to include this fixture directly.
+**`benchmark_time`**: Record wall clock time duration.
 
-**`benchmark_db_session`**: The SQLAlchemy session for a given test. Most tests shouldn't need to include this fixture directly.
+**`benchmark_memory`**: Record memory usage.
 
-**`test_run_benchmark`**: The SQLAlchemy ORM object for a given test. By including this fixutre in a test (or another fixture that includes it) you trigger the test being written to the benchmark database. This fixture includes data common to all tests, including python version, test name, and the test outcome.
+**`benchmark_task_durations`**: Record time spent computing, transferring data, spilling to disk, and deserializing data.
```

Comment:
```suggestion
**`benchmark_task_durations`**: Records time spent computing, transferring data, spilling to disk, and deserializing data.
```

Reply:
Given the argumentation regarding [PEP 257](https://peps.python.org/pep-0257/#one-line-docstrings) in my other comment, I'd prefer sticking with a coherent style.